frontend/src/theme/components/button.ts

```diff
@@ -113,6 +113,13 @@ const Button = {
       borderRadius: "md",
       fontSize: "xs",
     },
+    "secondary-filled": {
+      bg: "white",
+      color: "black",
+      padding: "8px 16px",
+      borderRadius: "md",
+      fontSize: "xs",
+    },
```

Comment:
Documentation is a little messy right now, but it'd be great to at least make a note of this in the [Notion documentation](https://www.notion.so/uwblueprintexecs/Chakra-Variants-17ac1b8da8a14e618ec081a1358c4072)